### PR TITLE
add new extension Bitbucket Search (Self Hosted)

### PR DESCRIPTION
**Pull request by yuercl**
## Description

new extension Bitbucket Search (Self Hosted)

## Screencast
screenshot-1
screenshot-4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @yuercl 👋 

Wouldn't it be smarter to make it a setting in the current Bitbucket search? I guess it has many of the same features and this way we only have to maintain one extension.

image

**Comment by yuercl** [comment]
> Hi @yuercl 👋
> 
> Wouldn't it be smarter to make it a setting in the current Bitbucket search? I guess it has many of the same features and this way we only have to maintain one extension.
> 
> image

Despite the similar name and ui interface, the apis and configuration behind the two are completely different.

File: extensions/bitbucket-search-self-hosted/src/components/repository/actions.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Action, Color, Icon } from "@raycast/api";
+import { PullRequestsList } from "./pullRequestsList";
+
+const ShowPullRequestsActions = (props: { repo: any }): JSX.Element => {
+  return (
+    <Action.Push
+      title="Show pull requests"
```
Comment (pernielsentikaer) [inline extensions/bitbucket-search-self-hosted/src/components/repository/actions.tsx:7]:
```suggestion
      title="Show Pull Requests"
```

File: extensions/bitbucket-search-self-hosted/src/components/repository/pullRequestsList.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    fetchPRs();
+  }, []);
+
+  if (state.error) {
+    showToast(Toast.Style.Failure, "Failed loading repositories", state.error.message);
+  }
+
+  return (
+    <List isLoading={!state.pullRequests && !state.error} searchBarPlaceholder="Search by name...">
+      <List.Section title="Open Pull Requests" subtitle={state.pullRequests?.length + ""}>
```
Comment (pernielsentikaer) [inline extensions/bitbucket-search-self-hosted/src/components/repository/pullRequestsList.tsx:51]:
```suggestion
      <List.Section title="Open Pull Requests" subtitle={`${state.pullRequests?.length} item${state.pullRequests?.length ? "s": ""}`}>
 
```

**Review by pernielsentikaer — COMMENTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* Update the date in the `CHANGELOG`

*  [metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots)

* Please allow me to image

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]

**Comment by yuercl** [comment]
Thanks

**Pull request merged**